static/app/views/settings/projectDebugFiles/externalSources/customRepositories/index.tsx

```diff
@@ -264,18 +321,66 @@ function CustomRepositories({
                     }
                   : repository
               }
+              hasFeature={hasFeature}
+              hasAccess={hasAccess}
               onDelete={handleDeleteRepository}
               onEdit={handleEditRepository}
             />
           ))
         )}
-      </PanelBody>
-    </Panel>
+      </HookedCustomSymbolSources>
+    );
+  }
+
+  return (
+    <Feature features={['symbol-sources']} organization={organization}>
```

Comment:
This is a redundant check, `custom-symbol-sources` will not be enabled if `symbol-sources` is off. I'd suggest to keep it simple just hide the entire UI when `symbol-sources` is off. The only case worth handling is `custom-symbol-sources`.

Reply:
The thing is that the UI uses the result of these checks somewhere else... For instance to show the getsentry `custom-symbol-sources` hook.

And I don't think that hidden things would make the user experience better, like for example, in case the user mistakenly disables some feature flag in the local environment. With these new changes, the user could have this feedback in the UI, which would be useful.

`symbol-sources` and `custom-symbol-sources` are enabled by default but it could be mistakenly disabled

Reply:
We had an offline conversation and concluded that it would be better to remove the `symbol-sources` checks from the UI since the feature flag will be removed soon.